### PR TITLE
Move create functions and behavior to dialog

### PR DESCRIPTION
**Pull request by fannieyan**

### CODE REVIEW

File: newIDE/app/src/AssetStore/ExtensionStore/ExtensionsSearchDialog.js [hunk 1]
```diff
@@ -93,6 +97,14 @@ export default function ExtensionsSearchDialog({
           fullHeight
           title={<Trans>Search for New Extensions</Trans>}
           actions={[
+            onCreateNew ? (
+              <RaisedButton
+                key="create-new"
+                onClick={onCreateNew}
+                label={<Trans>Create functions or behavior</Trans>}
```
[the remark this answers is not in the document]
Comment (4ian) [reply newIDE/app/src/AssetStore/ExtensionStore/ExtensionsSearchDialog.js:104]: `Create a new extension`? 

"Create functions or behavior[s]" is I think a bit long and not implying the "new" that someone might intuitively search on the screen when it comes to create something... new?

**Comment by tristanbob** [comment]
Thanks for working on this; I am super excited to see this minor (but long-standing) improvement!  

I think the primary purpose of this window is to search and install existing behaviors.  That is what it will be used for 99% of the time.  Therefore, I suggest making "Create new extensions" less visible and not the primary button.  Perhaps put it next to "Import Extensions"?

**Comment by AlexandreSi** [comment]
> Thanks for working on this; I am super excited to see this minor (but long-standing) improvement!
> 
> I think the primary purpose of this window is to search and install existing extensions. That is what it will be used for 99% of the time. Therefore, I suggest making "Create new extensions" less visible and not the primary button. Perhaps put it next to "Import Extensions"?

I agree with you, having it a secondary action seems to be a way to keep users on this dialog so that they can browse content

**Review by AlexandreSi — APPROVED** [review]
LGTM! Thanks for removing this 2 grey buttons that I have always misclicked

**Comment by ClementPasteau** [comment]
> Thanks for removing this 2 grey buttons that I have always misclicked

same 🙏

**Comment by fannieyan** [comment]
As per @tristanbob 's suggestion, I moved the button as secondary action

image

**Comment by 4ian** [comment]
Good call everyone here, the secondary button is indeed better from a UX point of view (because it's indeed a secondary task for this dialog, much like importing and because we don't want to the eye of the new/casual user who should look at extensions, not at this button) :)

**Pull request merged**